packages/code-du-travail-frontend/src/questionnaire/Components/Question/ShowInfo.tsx

```diff
@@ -28,7 +29,7 @@ export const ShowInfo = ({
           trackClickViewPageInfo();
           const destination = `/information/${slug}`;
           if (widgetMode) {
-            window.open(`${window.location.origin}${destination}`);
+            window.open(`${SITE_URL}${destination}`);
```

Comment:
Pour fixer l'erreur :
`Failed to read a named property 'origin' from 'Location': Blocked a frame with origin "https://code.travail.gouv.fr" from accessing a cross-origin frame.`

Reply:
```ts
export const SITE_URL =
  process.env.NEXT_PUBLIC_SITE_URL ?? `http://localhost:3000`;
```

ça revient au même qu'avec la variable d'environment non ? Car normalement la window est toujours défénie

Reply:
Je suis d'accord, on peut mettre l'adresse directement et ne pas demander au js de nous donner l'url que l'on connait déjà.